src/app/pages/dashboard/components/go-to-journal-card/go-to-journal-card.ts

```diff
@@ -9,10 +9,11 @@ import { JOURNAL_PAGE } from '../../../page-names.constants';
 })
 
 export class GoToJournalCardComponent {
-
+  @Output()
+  navigateClicked = new EventEmitter<string>();
   constructor(private router: Router) { }
```

Comment:
This should now be able to be removed, as well as via the spec if it is injected